ros_bridge_interface/src/ros_bridge_interface.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
+import rospy
+from std_msgs.msg import String
+import json
+import time
+
+
+class RosBridgeInterface:
+    def __init__(self):
+        rospy.init_node('ros_bridge_interface_server', anonymous=True)
+        self.interface_action_sub = rospy.Subscriber("ui/robot/call_action", String, self.interface_action_callback)
```

Comment:
Can we change the topic the interface node subscribes to to "nightingale_ui/user_input"? I think this makes more sense since it is listening to messages the tablet is sending for when the patient wants the robot to do something